docs/index.asciidoc

```diff
@@ -50,10 +51,80 @@ filter plugins.
   * Value type is <<array,array>>
   * There is no default value for this setting.
 
-A new clone will be created with the given type for each type in this list.
+ECS Compatibility disabled: a new clone will be created with a `type` of the given type in this list.
+
+ECS Compatibility enabled: a new clone will be created with a `tag` of the given type in this list.
+
+Example:
```

Comment:
@karenzone a bit hijacking here but where do we want general examples such as this?
pointing out how plugin enhances events in general, while also showcasing a difference in ECS vs non-ECS behavior (asking since I will also want to add examples on another plugin):
- start of the docs
- at the end of the page
- or right after the `<<plugins-{type}s-{plugin}-ecs_compatibility>>` secion

Reply:
@kares, thanks for asking. The best place for examples is where they make the most sense in context.  We always work to "_make information available to users at the time they need it, with minimal disruption in workflow_."  In this case, the examples make sense right after the <<plugins-{type}s-{plugin}-ecs_compatibility>> section.

I asked @kaisecheng Kaise to alphabetize the config options, but the examples got left behind. She had the example placement correct from the start. 

@kaisecheng, will you please relocate the ecs examples to be under the `ecs_compatibility` section?

Reply:
@karenzone The examples are in `clones` section with purpose. This section explains the relationship of `clones` => `type`. After ECS, it further explains `clones` => `tags`. As the wording is a bit difficult to understand, so I add examples following the `type` and `tags`. The examples are meant to educate user how to config `clones` and what they will see, but I understand it is confusing reviewers that they are for ecs

@kares suggested to not mentions `type` and `tags` in `ecs_compatibility` section. ✅ 
@karenzone suggested to move examples to `ecs_compatibility` section. ✅ 

@karenzone please have a look again if the workflow makes sense to you :)